tex/main/MarkovChain/expressions_from_pi/main.tex

```diff
@@ -328,3 +326,180 @@ \subsubsection{Overall Waiting Time}
+
+\begin{equation*}
+    p_s(u,v) = \frac{\min(v,C)\mu}{\lambda^o + \min(v,C)\mu}, \qquad
+    p_o(u,v) = \frac{\lambda^o}{\lambda^o + \min(v,C)\mu}
+\end{equation*}
+
+
+Having defined \( c(u,v) \) and \(S_b\) a formula for the blocking time that is
+expected to occur at each state can be given by:
+
+\begin{equation}\label{eq:blocking-time-at-each-state}
+    b(u,v) = 
+    \begin{cases} 
+        0, & \textbf{if } (u,v) \notin S_b \\
+        c(u,v) + b(u - 1, v), & \textbf{if } v = N = T\\
+        c(u,v) + b(u, v-1), & \textbf{if } v = N \neq T \\
+        c(u,v) + p_s(u,v) b(u-1, v) + p_o(u,v) b(u, v+1), & \textbf{if } u > 0 \textbf{ and } v = T \\
+        c(u,v) + p_s(u,v) b(u, v-1) + p_o(u,v) b(u, v+1), & \textbf{otherwise} \\
+    \end{cases}
+\end{equation}
+
+Unlike equations (\ref{eq:waiting-time-at-each-state-other}) and 
+(\ref{eq:waiting-time-at-each-state-ambulance}), equation 
+(\ref{eq:blocking-time-at-each-state}) cannot be solved recursively. 
+An algebraic approach will be used to solve this equation here. 
+By finding all equations \(b(u,v)\) for all states \((u,v)\) that belong in \(S_b\) 
+a system of linear equations arises where the unknown variables are all the \(b(u,v)\)
+terms.
+For instance, let us consider a Markov model where \(C=2, T=3, N=6, M=2\). 
+The Markov model and the equivalent equations are shown below. 
+The equations considered here are only the ones that correspond to the blocking 
+states.
+
+\begin{multicols*}{2}
+    \scalebox{0.68}{\input{MarkovChain/expressions_from_pi/example_model_2362/main.tex}}
+    \columnbreak
+    \vspace*{0cm} 
+    \begin{align*}
+        b(1,2) &= c(1,2) + p_o b(1,3) \\
+        b(1,3) &= c(1,3) + p_s b(1,2) + p_o b(1,4) \\
+        b(1,4) &= c(1,4) + b(1,3) \\
+        b(2,2) &= c(2,2) + p_s b(1,2) + p_o b(2,3) \\
+        b(2,3) &= c(2,3) + p_s b(2,2) + p_o b(1,4) \\
+        b(2,4) &= c(2,4) + b(2,3)
+    \end{align*}
+\end{multicols*}
+
+Additionally, the above equations can be transformed into the following linear system.
+
+\begin{equation}\label{eq:example-algebaric-approach-blocking-time}
+    \begin{pmatrix}
+        -1 & p_o & 0 & 0 & 0 & 0 \\ %(1,2)
+        p_s & -1 & p_o & 0 & 0 & 0 \\ %(1,3)
+        0 & 1 & -1 & 0 & 0 & 0 \\ %(1,4)
+        p_s & 0 & 0 & -1 & p_o & 0\\ %(2,2)
+        0 & 0 & 0 & p_s & -1 & p_o \\ %(2,3)
+        0 & 0 & 0 & 0 & 1 & -1 \\ %(2,4)
+    \end{pmatrix}
+    \begin{pmatrix}
+        b(1,2) \\
+        b(1,3) \\
+        b(1,4) \\
+        b(2,2) \\
+        b(2,3) \\
+        b(2,4) \\
+    \end{pmatrix}
+    = 
+    \begin{pmatrix}
+        -c(1,2) \\
+        -c(1,3) \\
+        -c(1,4) \\
+        -c(2,2) \\
+        -c(2,3) \\
+        -c(2,4) \\
+    \end{pmatrix}
+\end{equation}
+
+A more generalised form of equation \ref{eq:example-algebaric-approach-blocking-time}
+can thus be given for any value of \(C,T,N,M\) by:
+
+\begin{equation}\label{eq:general-algebaric-approach-blocking-time}
```

Comment:
So before this write out explicitly/rigorously the terms of the linear system and then go: `which will resemble:` followed by the `...` stuff.


(Let me know if that's not clear)

Reply:
So write down the actual generalised equations before the matrix? (similar to what I did with the example model)

Reply:
Yup :+1: